tools/model_tools/src/openvino/model_zoo/omz_quantizer.py

```diff
@@ -162,9 +162,9 @@ def main():
 
         pot_path = args.pot
         if pot_path is None:
-            if _common.get_package_path(args.python, 'pot'):
```

Comment:
should we preserve both old import way and new with try expect (atleast until openvino-dev has old import on pypi)?